src/lib/share/index.js

```diff
@@ -142,6 +148,20 @@ export function generateReceiveShareObject(code: string, amount: number, to: str
   return generateShareObject('Sending G$ via GoodDollar App', text, url)
 }
 
+export function generateReceiveShareText(code: string, amount: number, to: string, from: string): ShareObject {
```

Comment:
lots of duplicate code.
maybe just call generateREceiveShareObject and use the .text result

Reply:
done